spec/logstash/outputs/scalyr_integration_spec.rb

```diff
@@ -83,7 +83,8 @@
                 {
                   :error_class=>"Manticore::UnknownException",
                   :batch_num=>1,
-                  :message=>"java.lang.RuntimeException: Unexpected error: java.security.InvalidAlgorithmParameterException: the trustAnchors parameter must be non-empty",
+                  :message=>"Unexpected error: java.security.InvalidAlgorithmParameterException: the trustAnchors parameter must be non-empty",
+                  #:message=>"java.lang.RuntimeException: Unexpected error: java.security.InvalidAlgorithmParameterException: the trustAnchors parameter must be non-empty",
```

Comment:
Leftover?

Reply:
I left it there intentionally since exception changes some how - haven't tracked down yet why, but I think it may be related to jRuby version.